inc/wpcli/class-blocks-scaffold.php

```diff
@@ -0,0 +1,279 @@
+
+		// create editor assets.
+		$this->create_block_editor_assets();
+
+		// create FE assets.
+		$this->create_block_assets();
+
+		WP_CLI::success( $this->name . ' block created.' );
+	}
+
+	/**
+	 * Init file system.
+	 *
+	 * @since 2.0.0
+	 */
+	private function init_filesystem() {
+		// File system support.
+		global $wp_filesystem;
+		require_once ABSPATH . '/wp-admin/includes/file.php';
+		WP_Filesystem();
+
+		return $wp_filesystem;
+	}
+
+	/**
+	 * Create the block directory.
+	 *
+	 * @author Biplav Subedi <biplav.subedi@webdevstudios.com>
+	 * @since 2.0.0
+	 */
+	private function create_block_dir() {
+		$dir = ROOT_PATH . 'blocks/' . $this->name;
+
+		if ( ! $this->init_filesystem()->exists( $dir ) ) {
+			$this->init_filesystem()->mkdir( $dir, 0755 );
+		} else {
+			WP_CLI::error( 'Block directory already exists.', true );
+		}
+	}
+
+	/**
+	 * Create the block render file.
+	 *
+	 * @param array $args Block details.
+	 * @since 2.0.0
+	 * @author Biplav Subedi <biplav.subedi@webdevstudios.com>
+	 */
+	private function create_block_render_php( $args ) {
+		$local_file = ROOT_PATH . 'inc/wpcli/block-starter/block.php';
+		$content    = '';
+
+		if ( $this->init_filesystem()->exists( $local_file ) ) {
+			$content = $this->init_filesystem()->get_contents( $local_file );
+			$content = str_replace( 'wds', $args['namespace'], $content );
+			if ( ! $this->init_filesystem()->put_contents( ROOT_PATH . 'blocks/' . $this->name . '/' . $this->name . '.php', $content ) ) {
+				WP_CLI::error( 'ERROR :: Could not create a render file.', true );
+			}
+		} else {
+			WP_CLI::error( 'ERROR :: Could not create a render file.', true );
+		}
+
+	}
+
+	/**
+	 * Create the block json.
+	 *
+	 * @param array $args Block details.
+	 * @since 2.0.0
+	 * @author Biplav Subedi <biplav.subedi@webdevstudios.com>
+	 */
+	private function create_block_json( $args ) {
+		$local_file = ROOT_PATH . 'inc/wpcli/block-starter/block.json';
+		$content    = '';
+
+		if ( $this->init_filesystem()->exists( $local_file ) ) {
+			$content = $this->init_filesystem()->get_contents( $local_file );
+			$content = str_replace(
+				[
+					'{{name}}',
+					'{{title}}',
+					'{{description}}',
+					'{{icon}}',
+					'wds/',
+					'{{keyword}}',
+				],
+				[
+					$this->name,
+					$args['title'],
+					$args['desc'],
+					$args['icon'],
+					trailingslashit( $args['namespace'] ),
+					$args['keyword'],
+				],
+				$content
+			);
+		}
+
+		if ( ! $this->init_filesystem()->put_contents( ROOT_PATH . 'blocks/' . $this->name . '/block.json', $content ) ) {
+			WP_CLI::error( 'ERROR :: Could not create a block json file.', true );
+		}
+	}
+
+	/**
+	 * Create the block editor styles.
+	 *
+	 * @since 2.0.0
+	 * @author Biplav Subedi <biplav.subedi@webdevstudios.com>
+	 */
+	private function create_block_editor_assets() {
+		$assets_js  = ROOT_PATH . 'inc/wpcli/block-starter/editor.js';
+		$assets_css = ROOT_PATH . 'inc/wpcli/block-starter/editor.css';
```

Comment:
Remove the style references from this function. We only need the js.

Reply:
👍